amplipi/display/einkdisplay.py

```diff
@@ -1,29 +1,43 @@
 #!/usr/bin/env python3
 """ EInk display handler """
 
+import math
+import json
+import datetime
+import os
 
 from collections import namedtuple
 import sys
 import pathlib
 from time import sleep
-from typing import List, Optional
+from typing import List, Optional, Union, Tuple
 
 import socket
 import netifaces as ni
 from PIL import Image, ImageDraw, ImageFont
 from loguru import logger as log
+import requests
 
 from amplipi.display import epd2in13_V3
 from amplipi.display.common import DefaultPass, Display
+from amplipi.display.einkinterface import set_custom_eink_status
+from amplipi.display.einkinterface import status_filename
+from amplipi import models
+from amplipi import rt
 
-SysInfo = namedtuple('SysInfo', ['hostname', 'password', 'ip'])
+
+SysInfo = namedtuple('SysInfo', ['hostname', 'password', 'ip', 'status_code', "serial_number", "ext_count"])
+
+startup_msg = "Starting Up"
 
 
 class EInkDisplay(Display):
   """ Display system infomation on EInk Panel"""
```

Comment:
While we're in here...
```suggestion
  """ Display system information on EInk Panel"""
```